UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -610,7 +613,11 @@
     !if $(UNIVERSAL_PAYLOAD_FORMAT) == "ELF"
       UefiPayloadPkg/UefiPayloadEntry/UniversalPayloadEntry.inf
     !elseif $(UNIVERSAL_PAYLOAD_FORMAT) == "FIT"
-      UefiPayloadPkg/UefiPayloadEntry/FitUniversalPayloadEntry.inf

```

Comment:
This code also makes assumption that there is only x86 arch in the world. It should have else if statements. Just wanted to draw attention or in future I will have to modify this and submit patch anyways.

Reply:
we are looking into this, ideally most of the lib and entry module are arch agnostic, just some small lib may be arch/platform specific (for example FdtCustomNodeParserLib)